offload-build.in

```diff
@@ -108,7 +115,11 @@ mapfile -t files < <(
                 printf "%s\n" "" "-> build complete" &&
                 printf "\t%s\n" "$temp"/*
             } >&2 &&
-            makepkg --packagelist
+            makepkg_config='"${makepkg_config@Q}"' &&
+            if [[ -f '"${makepkg_config@Q/local\//}"' ]]; then
+                makepkg_config='"${makepkg_config@Q/local\//}"'
```

Comment:
So this hardcodes /usr/share for the buildserver anyway, and just uses local machine introspection to detect if there is a makepkg-repo-arch.conf? I'm not sure how that corresponds to the concern I initially raised.

This also breaks quite badly if the local machine is running, idk, nixos with devtools at /nix/store/${some_long_hash}-devtools-${version}/ as the prefix, since it binds together the local machine structure with the remote machine structure.

Reply:
well we can also hardcode /usr/share and purely run on the remote, but not sure of we wanna have a second set of checks to support remotes that have an installation in /usr/local. at some point we need to make some assertions how to resolve such locations.

Reply:
pacman uses `pkg-config libmakepkg --variable=libmakepkgdir`, though that is intended for use in build environments, not runtime environments.